deduplication/src/main/java/gov/cdc/nbs/deduplication/algorithm/AlgorithmController.java

```diff
@@ -41,4 +54,26 @@ public Map<String, List<Pass>> getMatchingConfiguration() {
     public void updateAlgorithm(@RequestBody MatchingConfigRequest request) {
         algorithmService.updateDibbsConfigurations(request);
     }
+    @GetMapping("/export-configuration")
+    public ResponseEntity<Resource> exportConfiguration() throws IOException {
+        List<Pass> passes = algorithmService.getMatchingConfiguration();
+
+        // Generate timestamped filename
+        String timestamp = LocalDateTime.now().format(DateTimeFormatter.ofPattern("yyyyMMdd_HHmmss"));
+        String fileName = "record_linker_config_" + timestamp + ".json";
+        String tempDir = System.getProperty("java.io.tmpdir");
+        Path filePath = Paths.get(tempDir, fileName);
```

Comment:
Question: Do we need to save the file to the tmp directory? Can we just convert the config to bytes and create an `InputStreamResource`